src/identity/Identity.ts

```diff
@@ -155,6 +406,19 @@ export default class Identity extends PublicIdentity {
     )
   }
 
+  /**
+   * @description Signs a submittable extrinsic, an extrinsic are pieces of information not from within the state of the chain.
```

Comment:
Not sure about the grammar 🙃

Reply:
I have updated, hopefully, should be better.